From: Srinivas Kandagatla <srinivas.kandagatla@linaro.org>
Date: Wed, 26 Jul 2017 11:34:48 +0200
Subject: nvmem: include linux/err.h from header
Patch-mainline: v4.14-rc1
Git-commit: 4da69f49e73e3c79f079de62ea940cabbbf15ae7
References: FATE#323952, FATE#322506

missing err.h header file can cause simillar errors like below
in some configurations.
"error: implicit declaration of function 'ERR_PTR'
[-Werror=implicit-function-declaration]"

This adds the missing include to ensure we can always include
the header.

Signed-off-by: Srinivas Kandagatla <srinivas.kandagatla@linaro.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: Johannes Thumshirn <jthumshirn@suse.de>
---
 include/linux/nvmem-consumer.h |    3 +++
 1 file changed, 3 insertions(+)

--- a/include/linux/nvmem-consumer.h
+++ b/include/linux/nvmem-consumer.h
@@ -12,6 +12,9 @@
 #ifndef _LINUX_NVMEM_CONSUMER_H
 #define _LINUX_NVMEM_CONSUMER_H
 
+#include <linux/err.h>
+#include <linux/errno.h>
+
 struct device;
 struct device_node;
 /* consumer cookie */
